src/Messages.jl

```diff
@@ -609,7 +609,23 @@ function Base.show(io::IO, m::Message)
     end
     println(io, typeof(m), ":")
     println(io, "\"\"\"")
-    writeheaders(io, m)
+
+    # Mask sensitive header values
+    # The following headers values contain sensitive information that
+    # we don't want to show
+    # - Authorization
+    # - Cookie
+    # - Set-Cookie (in response)
+    # We will show "**********" instead
+    ioh = IOBuffer()
+    writeheaders(ioh, m)
+    header_str = String(take!(ioh))
```

Comment:
```suggestion
    header_str = sprint(writeheaders, m)
```
should be equivalent and a bit shorter.